npc/re/jobs/novice/academy.txt

```diff
@@ -6634,71 +6635,71 @@ iz_ac01_b,103,89,3	duplicate(ac_board)	[Great Hall]#ac_b	2_BULLETIN_BOARD
-iz_ac02_d,109,140,3	duplicate(ac_board)	[Ninja Classroom]#ac_d	2_BULLETIN_BOARD
-
-iz_ac02,98,110,5	duplicate(ac_board)	[Archer Classroom]#ac	2_BULLETIN_BOARD
-iz_ac02_a,98,110,5	duplicate(ac_board)	[Archer Classroom]#ac_a	2_BULLETIN_BOARD
-iz_ac02_b,98,110,5	duplicate(ac_board)	[Archer Classroom]#ac_b	2_BULLETIN_BOARD
-iz_ac02_c,98,110,5	duplicate(ac_board)	[Archer Classroom]#ac_c	2_BULLETIN_BOARD
-iz_ac02_d,98,110,5	duplicate(ac_board)	[Archer Classroom]#ac_d	2_BULLETIN_BOARD
-
-iz_ac02,109,110,3	duplicate(ac_board)	[Mage Classroom]#ac	2_BULLETIN_BOARD
-iz_ac02_a,109,110,3	duplicate(ac_board)	[Mage Classroom]#ac_a	2_BULLETIN_BOARD
-iz_ac02_b,109,110,3	duplicate(ac_board)	[Mage Classroom]#ac_b	2_BULLETIN_BOARD
-iz_ac02_c,109,110,3	duplicate(ac_board)	[Mage Classroom]#ac_c	2_BULLETIN_BOARD
-iz_ac02_d,109,110,3	duplicate(ac_board)	[Mage Classroom]#ac_d	2_BULLETIN_BOARD
-
-iz_ac02,98,80,5	duplicate(ac_board)	[Taekwon Classroom]#2	2_BULLETIN_BOARD
-iz_ac02_a,98,80,5	duplicate(ac_board)	[Taekwon Classroom]#2_a	2_BULLETIN_BOARD
-iz_ac02_b,98,80,5	duplicate(ac_board)	[Taekwon Classroom]#2_b	2_BULLETIN_BOARD
-iz_ac02_c,98,80,5	duplicate(ac_board)	[Taekwon Classroom]#2_c	2_BULLETIN_BOARD
-iz_ac02_d,98,80,5	duplicate(ac_board)	[Taekwon Classroom]#2_d	2_BULLETIN_BOARD
-
-iz_ac02,109,80,3	duplicate(ac_board)	[Gunslinger Classroom]#0	2_BULLETIN_BOARD
-iz_ac02_a,109,80,3	duplicate(ac_board)	[Gunslinger Classroom]#a	2_BULLETIN_BOARD
-iz_ac02_b,109,80,3	duplicate(ac_board)	[Gunslinger Classroom]#b	2_BULLETIN_BOARD
-iz_ac02_c,109,80,3	duplicate(ac_board)	[Gunslinger Classroom]#c	2_BULLETIN_BOARD
-iz_ac02_d,109,80,3	duplicate(ac_board)	[Gunslinger Classroom]#d	2_BULLETIN_BOARD
-
-iz_ac02,98,50,5	duplicate(ac_board)	[Swordman Classroom]#2	2_BULLETIN_BOARD
-iz_ac02_a,98,50,5	duplicate(ac_board)	[Swordman Classroom]#2_a	2_BULLETIN_BOARD
-iz_ac02_b,98,50,5	duplicate(ac_board)	[Swordman Classroom]#2_b	2_BULLETIN_BOARD
-iz_ac02_c,98,50,5	duplicate(ac_board)	[Swordman Classroom]#2_c	2_BULLETIN_BOARD
-iz_ac02_d,98,50,5	duplicate(ac_board)	[Swordman Classroom]#2_d	2_BULLETIN_BOARD
-
-iz_ac02,109,50,3	duplicate(ac_board)	[Lounge]#ac	2_BULLETIN_BOARD
-iz_ac02_a,109,50,3	duplicate(ac_board)	[Lounge]#ac_a	2_BULLETIN_BOARD
-iz_ac02_b,109,50,3	duplicate(ac_board)	[Lounge]#ac_b	2_BULLETIN_BOARD
-iz_ac02_c,109,50,3	duplicate(ac_board)	[Lounge]#ac_c	2_BULLETIN_BOARD
-iz_ac02_d,109,50,3	duplicate(ac_board)	[Lounge]#ac_d	2_BULLETIN_BOARD
-
-iz_ac02,107,180,3	duplicate(ac_board)	[Principal's Office]#a	2_BULLETIN_BOARD
-iz_ac02_a,107,180,3	duplicate(ac_board)	[Principal's Office]#aa	2_BULLETIN_BOARD
-iz_ac02_b,107,180,3	duplicate(ac_board)	[Principal's Office]#ab	2_BULLETIN_BOARD
-iz_ac02_c,107,180,3	duplicate(ac_board)	[Principal's Office]#ac	2_BULLETIN_BOARD
-iz_ac02_d,107,180,3	duplicate(ac_board)	[Principal's Office]#ad	2_BULLETIN_BOARD
+iz_ac02,202,46,5	duplicate(ac_board)	[Swordman Classroom]#2	2_BULLETIN_BOARD
```

Comment:
Can you please keep the original order of npcs so that only the coordinate changes are seen?
Thank you.

Reply:
Oh, for sure.
I just did that to make easier to me to know where I put the boards, sorry.

Reply:
[academy.txt](https://github.com/rathena/rathena/files/6850512/academy.txt)
fix review changes.